sensor/kubernetes/clusterhealth/updater.go

```diff
@@ -157,6 +163,44 @@ func (u *updaterImpl) getAdmissionControlInfo() *storage.AdmissionControlHealthI
 	return &result
 }
 
+func (u *updaterImpl) getLocalScannerInfo() *storage.LocalScannerHealthInfo {
+	if !env.LocalImageScanningEnabled.BooleanSetting() {
+		return nil
+	}
+	result := storage.LocalScannerHealthInfo{}
+	// Local Scanner deployment is looked up in the same namespace as Sensor because that is how they should be deployed.
+	localScanner, err := u.client.AppsV1().Deployments(u.namespace).Get(u.ctx(), localScannerDeploymentName, metav1.GetOptions{})
+	if err != nil {
+		err = errors.Wrap(err, fmt.Sprintf("unable to find local scanner deployments in namespace %q", u.namespace))
+		result.StatusErrors = append(result.StatusErrors, fmt.Sprintf("unable to find local scanner deployments in namespace %q: %v", u.namespace, err))
```

Comment:
```suggestion
		err = errors.Wrap(err, fmt.Sprintf("unable to find scanner deployment in namespace %q", u.namespace))
		result.StatusErrors = append(result.StatusErrors, fmt.Sprintf("unable to find scanner deployment in namespace %q: %v", u.namespace, err))
```
Similar below for the DB case.

Reply:
Done.